yarn.lock

```diff
@@ -1765,6 +1765,11 @@
   resolved "https://registry.yarnpkg.com/@schemastore/tsconfig/-/tsconfig-0.0.9.tgz#77a1a1fb55eb879391aa78db3154dbcc4d93948b"
```

Comment:
wow, this is a lot of dependencies. wild.

Reply:
Yeah I know - the world of 1 liner modules...